docker-compose-developer.yml

```diff
@@ -59,9 +59,6 @@ services:
     volumes:
       - ./grafana-plugin:/etc/app
       - node_modules_dev:/etc/app/node_modules
-      # https://stackoverflow.com/a/60456034
-      # see the explaination above that uses the $ENTERPRISE_ENGINE env var
-      - ${ENTERPRISE_FRONTEND:-/dev/null}:${ENTERPRISE_FRONTEND_VOLUME_MOUNT_DEST_DIR:-/tmp/empty:ro}
```

Comment:
this is no longer needed since we moved the frontend enterprise stuff to this repo